demoscene/utils/groklinks.py

```diff
@@ -1531,14 +1531,26 @@ def __unicode__(self):
         (domain, name) = self.param.split('/')
         return u"https://%s.bandcamp.com/track/%s" % (domain, name)
 
+def TwitchChannel(BaseUrl):
+    canonical_format = "https://twitch.tv/%s" # Channel name
+    tests = [
+        # Use (\w+) as Twitch does not accept `-` during registration
+        querystring_match(r'https?://m\.twitch\.tv/(\w+)', re.I), # Mobile version
```

Comment:
This should be `regex_match` rather than `querystring_match` as we're matching part of the URL path, not a `?foo=bar` query parameter.